website/source/docs/builders/oneandone.html.md

```diff
@@ -0,0 +1,58 @@
+---
+description: |
+    The 1&1 builder is able to create images for 1&1 cloud.
+layout: docs
+page_title: 1&1 Builder
+...
+
+# 1&1 Builder
+
+Type: `oneandone`
+
+The 1&1 Builder is able to create virtual machines for [One](https://www.1and1.com/).
+
+## Configuration Reference
+
+There are many configuration options available for the builder. They are
+segmented below into two categories: required and optional parameters. Within
+each category, the available configuration keys are alphabetized.
+
+In addition to the options listed here, a
+[communicator](/docs/templates/communicator.html) can be configured for this
+builder.
+
+### Required
+
+-   `source_image_name` (string) - 1&1 Server Appliance name of type `IMAGE`. Defaults to `ubuntu1604-64std`
+
+-   `token` (string) - 1&1 REST API Token. This can be specified via environment variable `ONEANDONE_TOKEN`
+
+### Optional
+
+-   `disk_size` (string) - Amount of disk space for this image in GB. Defaults to "50"
+
+-   `image_password` (string) - Password for the server images.
+
+-   `image_name` (string) - Resulting image. If "image_name" is not provided Packer will generate it
+
+-   `timeout` (string) - An approximate limit on how long Packer will continue making status requests while waiting for the build to complete. Default value "600".
```

Comment:
If we are using a string here I think it's better to take a `time.Duration` and convert it to seconds or whatever the API expects.